smart_card_connector_app/src/create_app_manifest.py

```diff
@@ -162,7 +162,7 @@ def main():
       dest='target_file')
   args_parser.add_argument(
       '--enable-condition',
-      nargs='+',
```

Comment:
Turns out, "nargs" didn't work properly with multiple arguments - it was creating a list, but always of size 1, regardless of the number of passed arguments o_O
"action=append" seems to do what we want - build a list of all conditions that were specified in the command line.